rootdir/init.shinano.pwr.rc

```diff
@@ -44,28 +44,25 @@ on boot
     chmod 664 /sys/devices/system/cpu/cpu2/online
     chmod 664 /sys/devices/system/cpu/cpu3/online
 
-on property:service.bootanim.exit=1
+on property:sys.boot_completed=1
```

Comment:
Why not doing like on KK and set that from somewhere else? Makes less changes to maintain regarding upstream :)